types/summernote/index.d.ts

```diff
@@ -1,7 +1,8 @@
 // Type definitions for Summernote 0.8
 // Project: https://github.com/summernote/summernote#readme
 // Definitions by: Wouter Staelens <https://github.com/wstaelens>
-//                 Denny Harijanto <https://github.com/nusantara-cloud>
+//                 Denny Harijanto <https://github.com/nusantara-cloud>,
```

Comment:
Once the owner list is line-delimited, commas are unnecessary.

Reply:
Whoops, dunno how I did this. My bad